infra/aws/terraform/management-account/accounts.tf

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+resource "aws_organizations_account" "organisation_security" {
+  name                       = "k8s-infra-organisation-security"
+  email                      = "k8s-infra-aws-admins+security@kubernetes.io"
+  iam_user_access_to_billing = "ALLOW"
+  parent_id                  = aws_organizations_organizational_unit.security.id
+
+  lifecycle {
+    ignore_changes = [
+      email,
+      iam_user_access_to_billing,
+      name,
```

Comment:
Why ignore `name` changes?

Reply:
Name changes is disruptive change that trigger account recreation.

Reply:
Cool. I'd comment to that effect so it's clear.

Reply:
Moved to the module.